docs/en/observability/whats-new.asciidoc

```diff
@@ -0,0 +1,15 @@
+[[whats-new]]
+= What's new in {minor-version}
+
+Here are the highlights of what's new and improved in {minor-version}.
+
+coming[8.0.0]
+
+// Other versions:
+// {observability-guide-all}/7.10/whats-new[7.10] |
+// https://www.elastic.co/blog/whats-new-elastic-observability-7-9-0-unified-agent-kpi-overview-dashboard[7.9] |
+// https://www.elastic.co/blog/elastic-observability-7-8-0-released[7.8] |
+// https://www.elastic.co/blog/elastic-observability-7-7-0-released[7.7] |
+// https://www.elastic.co/blog/elastic-observability-7-6-0-released[7.6] |
+// https://www.elastic.co/blog/elastic-observability-7-5-0-released[7.5] |
+// https://www.elastic.co/blog/elastic-observability-update-7-4-0[7.4]
```

Comment:
I think these links to other releases are helpful, even if they do link to blogs. I could only find release blogs back to 7.4.